### PR TITLE
Fix template directory structure and test references

### PR DESCRIPTION
**Pull request by vasukulkarni**
Signed-off-by: Vasu Kulkarni <vasu@redhat.com>

### CODE REVIEW

**Comment by vasukulkarni** [comment]
```
tests/test_add_mds_to_cluster.py::TestCephFilesystemCreation::test_cephfilesystem_creation 
--------------------------------------------------------------------------------------------- live log setup ---------------------------------------------------------------------------------------------
19:07:40 - MainThread - resources.ocs - INFO - Adding CephFilesystem with name cephfs-test-0619074052
19:07:40 - MainThread - utility.utils - INFO - Executing command: oc -n openshift-storage --kubeconfig /tmp/ocs-ci-vasukulk-68203/auth/kubeconfig create -f /var/folders/nt/g_hzgqps1q52bc1wbfy3sl_c0000gn/T/CephFilesystem71vgov36 -o yaml
19:07:41 - MainThread - utility.utils - INFO - Executing command: oc -n openshift-storage --kubeconfig /tmp/ocs-ci-vasukulk-68203/auth/kubeconfig get CephFilesystem cephfs-test-0619074052 -o yaml
19:07:41 - MainThread - utility.utils - INFO - Executing command: oc -n openshift-storage --kubeconfig /tmp/ocs-ci-vasukulk-68203/auth/kubeconfig get Pod --selector=app=rook-ceph-mds -o yaml
19:07:45 - MainThread - utility.utils - INFO - Executing command: oc -n openshift-storage --kubeconfig /tmp/ocs-ci-vasukulk-68203/auth/kubeconfig get Pod --selector=app=rook-ceph-mds -o yaml
19:07:48 - MainThread - utility.utils - INFO - Executing command: oc -n openshift-storage --kubeconfig /tmp/ocs-ci-vasukulk-68203/auth/kubeconfig get Pod --selector=app=rook-ceph-mds -o yaml
19:07:52 - MainThread - utility.utils - INFO - Executing command: oc -n openshift-storage --kubeconfig /tmp/ocs-ci-vasukulk-68203/auth/kubeconfig get Pod --selector=app=rook-ceph-mds -o yaml
19:07:55 - MainThread - utility.utils - INFO - Executing command: oc -n openshift-storage --kubeconfig /tmp/ocs-ci-vasukulk-68203/auth/kubeconfig get Pod --selector=app=rook-ceph-mds -o yaml
19:07:59 - MainThread - utility.utils - INFO - Executing command: oc -n openshift-storage --kubeconfig /tmp/ocs-ci-vasukulk-68203/auth/kubeconfig get Pod --selector=app=rook-ceph-mds -o yaml
19:08:02 - MainThread - utility.utils - INFO - Executing command: oc -n openshift-storage --kubeconfig /tmp/ocs-ci-vasukulk-68203/auth/kubeconfig get Pod --selector=app=rook-ceph-mds -o yaml
19:08:03 - MainThread - utility.utils - INFO - Executing command: oc -n openshift-storage --kubeconfig /tmp/ocs-ci-vasukulk-68203/auth/kubeconfig get Pod --selector=app=rook-ceph-mds -o yaml
--------------------------------------------------------------------------------------------- live log call ----------------------------------------------------------------------------------------------
19:08:04 - MainThread - utility.utils - INFO - Executing command: oc -n openshift-storage --kubeconfig /tmp/ocs-ci-vasukulk-68203/auth/kubeconfig apply -f /var/folders/nt/g_hzgqps1q52bc1wbfy3sl_c0000gn/T/CephFilesystem_7cufugc
19:08:05 - MainThread - utility.utils - ERROR - CMD error:: Warning: oc apply should be used on resource created by either oc create --save-config or oc apply
19:08:05 - MainThread - utility.utils - INFO - Executing command: oc -n openshift-storage --kubeconfig /tmp/ocs-ci-vasukulk-68203/auth/kubeconfig get Pod --selector=app=rook-ceph-mds -o yaml
19:08:08 - MainThread - utility.utils - INFO - Executing command: oc -n openshift-storage --kubeconfig /tmp/ocs-ci-vasukulk-68203/auth/kubeconfig get Pod --selector=app=rook-ceph-mds -o yaml
19:08:12 - MainThread - utility.utils - INFO - Executing command: oc -n openshift-storage --kubeconfig /tmp/ocs-ci-vasukulk-68203/auth/kubeconfig get Pod --selector=app=rook-ceph-mds -o yaml
19:08:16 - MainThread - utility.utils - INFO - Executing command: oc -n openshift-storage --kubeconfig /tmp/ocs-ci-vasukulk-68203/auth/kubeconfig get Pod --selector=app=rook-ceph-mds -o yaml
19:08:19 - MainThread - utility.utils - INFO - Executing command: oc -n openshift-storage --kubeconfig /tmp/ocs-ci-vasukulk-68203/auth/kubeconfig get Pod --selector=app=rook-ceph-mds -o yaml
19:08:23 - MainThread - utility.utils - INFO - Executing command: oc -n openshift-storage --kubeconfig /tmp/ocs-ci-vasukulk-68203/auth/kubeconfig get Pod --selector=app=rook-ceph-mds -o yaml
19:08:27 - MainThread - utility.utils - INFO - Executing command: oc -n openshift-storage --kubeconfig /tmp/ocs-ci-vasukulk-68203/auth/kubeconfig get Pod --selector=app=rook-ceph-mds -o yaml
19:08:27 - MainThread - utility.utils - INFO - Executing command: oc -n openshift-storage --kubeconfig /tmp/ocs-ci-vasukulk-68203/auth/kubeconfig get Pod --selector=app=rook-ceph-mds -o yaml
PASSED                                                                                                                                                                                             [100%]
------------------------------------------------------------------------------------------- live log teardown --------------------------------------------------------------------------------------------
19:08:28 - MainThread - utility.utils - INFO - Executing command: oc -n openshift-storage --kubeconfig /tmp/ocs-ci-vasukulk-68203/auth/kubeconfig delete CephFilesystem cephfs-test-0619074052 --wait=true


============================================================================================ warnings summary ============================================================================================
venv2/lib/python3.7/site-packages/_pytest/config/__init__.py:784
  /Users/vasukulkarni/ocs-ci/venv2/lib/python3.7/site-packages/_pytest/config/__init__.py:784: PytestAssertRewriteWarning: Module already imported so cannot be rewritten: utility
    self._mark_plugins_for_rewrite(hook)

-- Docs: https://docs.pytest.org/en/latest/warnings.html
========================================================================== 1 passed, 7 deselected, 1 warnings in 48.91 seconds ===========================================================================

tests/test_pv_creation.py::TestPvCreation::test_pv_creation 
--------------------------------------------------------------------------------------------- live log call ----------------------------------------------------------------------------------------------
19:07:21 - MainThread - tests.test_pv_creation - INFO - Creating new Persistent Volume
19:07:21 - MainThread - utility.utils - INFO - Executing command: oc -n openshift-storage --kubeconfig /tmp/ocs-ci-vasukulk-68203/auth/kubeconfig create -f test.yaml -o yaml
19:07:22 - MainThread - utility.utils - INFO - Executing command: oc -n openshift-storage --kubeconfig /tmp/ocs-ci-vasukulk-68203/auth/kubeconfig get PersistentVolume my-pv1 -o yaml
19:07:23 - MainThread - utility.utils - INFO - Executing command: oc -n openshift-storage --kubeconfig /tmp/ocs-ci-vasukulk-68203/auth/kubeconfig get PersistentVolume my-pv1 -o yaml
19:07:23 - MainThread - tests.test_pv_creation - INFO - PV my-pv1 exist
PASSED                                                                                                                                                                                             [100%]
------------------------------------------------------------------------------------------- live log teardown --------------------------------------------------------------------------------------------
19:07:23 - MainThread - tests.test_pv_creation - INFO - Deleting the Persistent Volume my-pv1
19:07:23 - MainThread - utility.utils - INFO - Executing command: oc -n openshift-storage --kubeconfig /tmp/ocs-ci-vasukulk-68203/auth/kubeconfig delete -f test.yaml --wait=true
19:07:24 - MainThread - utility.utils - INFO - Executing command: oc -n openshift-storage --kubeconfig /tmp/ocs-ci-vasukulk-68203/auth/kubeconfig get PersistentVolume my-pv1 -o yaml
19:07:25 - MainThread - utility.utils - ERROR - CMD error:: Error from server (NotFound): persistentvolumes "my-pv1" not found
19:07:25 - MainThread - tests.test_pv_creation - INFO - PV my-pv1 doesn't exist


============================================================================================ warnings summary ============================================================================================

```

**Comment by vasukulkarni** [comment]
Haven't touched the deployment templates yet

File: tests/test_pv_creation.py [hunk 1]
```diff
@@ -15,7 +15,7 @@
 
 log = logging.getLogger(__name__)
 
-PV_YAML = os.path.join("templates/ocs-deployment", "PersistentVolume.yaml")
+PV_YAML = f"{TEMPLATE_PV_PVC_DIR}/PersistentVolume.yaml"
```
Comment (fbalak) [inline tests/test_pv_creation.py]:
```suggestion
PV_YAML = os.path.join(TEMPLATE_PV_PVC_DIR, "PersistentVolume.yaml")
```

File: ocs/defaults.py [hunk 2]
```diff
@@ -18,7 +18,12 @@
 
 THIS_DIR = os.path.dirname(os.path.abspath(__file__))
 TOP_DIR = os.path.dirname(THIS_DIR)
-TEMPLATE_DIR = os.path.join(TOP_DIR, "templates/ocs-deployment/")
+TEMPLATE_DIR = f"{TOP_DIR}/templates/"
```
Comment (fbalak) [inline ocs/defaults.py]: I wouldn't replace `os.path.join` with f-string. https://stackoverflow.com/a/13944874

File: tests/test_pv_creation.py (hunk 1 above)
Comment (clacroix12) [reply tests/test_pv_creation.py]: @vasukulkarni We should use `os.path.join()` here as well

**Pull request merged**